src/js/pouchdb-node.js

```diff
@@ -0,0 +1,169 @@
+/*
+
+    A node-specific extension of gpii.pouch that adds additional abilities, such as loading data from package-specific
+    paths.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+var fs   = require("fs");
+var os   = require("os");
+var path = require("path");
+require("../../");
+
+fluid.registerNamespace("gpii.pouch.node");
+
+/**
+ *
+ * Create our working directory if it doesn't already exist.
+ *
+ * @param that - The component itself.
+ *
+ */
+gpii.pouch.node.initDir = function (that) {
```

Comment:
We should discuss further, as this is deeply entangled with the "do I already have data" check below.